scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -563,6 +589,8 @@ void BPF_STRUCT_OPS(layered_dispatch, s32 cpu, struct task_struct *prev)
 		if (layers[idx].preempt && scx_bpf_consume(idx))
 			return;
 
+	scx_bpf_consume(HI_FALLBACK_DSQ);
```

Comment:
What's the rationale for not checking the return value here and returning? Elsewhere you are only ever consuming a single task which makes this all a strict priority, but here you're falling through.

Reply:
Yeah, that's wrong. Will fix.